subsys/llext/CMakeLists.txt

```diff
@@ -17,4 +17,8 @@ if(CONFIG_LLEXT)
     fs_loader.c
 	)
   zephyr_library_sources_ifdef(CONFIG_LLEXT_SHELL shell.c)
+
+  if(CONFIG_RISCV AND CONFIG_USERSPACE)
+	message(WARNING "Running LLEXT extensions from user-space threads on RISC-V is not supported!")
```

Comment:
Should this be a FATAL_ERROR, or can you still run llexts in supervisor mode?
Please add a RISCV section in `doc/services/llext` with these details / caveats.

Reply:
So far, I have not encountered any issues running llexts in supervisor mode.
I added this warning because I encountered an instruction access fault in the `llext.simple.readonly_mpu` scenario in the llext test suite, specifically when attempting to run the llext in a user thread. This appears to be caused by incorrect configuration of the PMP, which is probably unrelated to the relocations.